presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,68 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for regular expression functions when using the Joni regex engine.
+* Fix query failures caused by Parquet predicate pushdown for smallint and tinyint types.
+* Fix query failures when view columns were fully qualified with catalog and/or schema.
+* Fix query failures when creating and reading an unpartitioned Hive table within the same transaction.
+* Fix an issue that would disable a performance optimization if two tables with mismatching
+  but compatible bucket counts are joined and one of the table has a `$bucket` filter.
+* Fix an issue that may cause procedure calls to fail if no queries were run after the server started up.
+* Fix an issue where the analyze properties would remain in the ``system.metadata.analyze_properties`` table even after the connector was removed.
```

Comment:
What does `analyze properties` mean here? This reads weirdly.

Reply:
@rschlussel

Reply:
Changed to 
```
Fix an issue where the properties supported by the ``ANALYZE`` statement for a given connector would remain in the ``system.metadata.analyze_properties`` table
  even after the connector was removed.
```